Sources/MapboxDirections/TollCollection.swift

```diff
@@ -39,13 +39,15 @@ public struct TollCollection: Codable, Equatable, ForeignMemberContainer {
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         type = try container.decode(CollectionType.self, forKey: .type)
+        name = try container.decodeIfPresent(String.self, forKey: .name)
```

Comment:
I think #669 caused the coding round-tripping tests to get stricter about properties that fail to round-trip. That’s a good thing!

/cc @Udumft @jill-cardamon